src/UnitConverter/Unit/Time/TimeUnit.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * This file is part of the jordanbrauer/unit-converter PHP package.
+ *
+ * @copyright 2017 Jordan Brauer <jbrauer.inc@gmail.com>
+ * @license MIT
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types = 1);
+
+namespace UnitConverter\Unit\Time;
+
+use UnitConverter\Measure;
+use UnitConverter\Unit\{
+    AbstractUnit, Volume\Second
+};
```

Comment:
This can reduced to simply,
```php
use UnitConverter\Measure;
use UnitConverter\Unit\AbstractUnit,
```
Since Second.php lives in the same namespace as TimeUnit, we don't need to specify a namespace path to import the class – it can simply be used _as is_, by all classes within the same namespace (this goes for all unit classes above that are importing Time\TimeUnit as well) 😃